docs/docs/policies.mdx

```diff
@@ -396,6 +398,268 @@ for details.
+has taken a user turn that is not well represented by training stories and needs a review.
+
+To reduce false warnings, `IntentTEDPolicy` has two mechanisms in place at inference time:
+
+1. `IntentTEDPolicy`'s [priority](./policies.mdx#policy-priority) is intentionally kept lower than all
+[rule based policies](./policies.mdx#rule-based-policies). This ensures that if an `action_unlikely_intent`
+is predicted for the last intent, but there is a rule that can be triggered to predict
+the next action, the ensemble of policies is not affected by it and the winning prediction is
+still the one made by `RulePolicy`.
+
+2. `IntentTEDPolicy` does not predict an `action_unlikely_intent` if the last predicted intent
+isn't present in any of the training stories.
+
+As mentioned earlier, while `TEDPolicy` learns the next best action, `IntentTEDPolicy` learns the
+next likely intent that a user can express. Given a conversation context, there can only be a single correct action,
+whereas there can be multiple intents that are likely to occur in the same conversation context. Hence,
+`IntentTEDPolicy` uses a multi-label cross entropy loss function during training.
+
+#### Prediction of `action_unlikely_intent`
+
+After training the internal ML model, `IntentTEDPolicy` does the following for each intent:
+
+1. Collect conversation contexts from training stories for which the corresponding intent is not the correct intent.
+2. Run these conversation contexts through the trained ML model and capture the predicted scores for the corresponding intent.
+3. Compute multiple quantiles
+
+**Configuration:**
+
+You can pass configuration parameters to the `IntentTEDPolicy` using the `config.yml` file.
+If you want to fine-tune model's performance, start by modifying the following parameters:
+
+* `epochs`:
+  This parameter sets the number of times the algorithm will see the training data (default: `1`).
+  One `epoch` is equals to one forward pass and one backward pass of all the training examples.
+  Sometimes the model needs more epochs to properly learn.
+  Sometimes more epochs don't influence the performance.
+  The lower the number of epochs the faster the model is trained.
+  Here is how the config would look like:
+
+  ```yaml-rasa title="config.yml"
+  policies:
+  - name: IntentTEDPolicy
+    epochs: 200
+  ```
+
+* `max_history`:
+  This parameter controls how much dialogue history the model looks at to decide which
+  action to take next. Default `max_history` for this policy is `None`,
+  which means that the complete dialogue history since session restart is taken into
+  account. If you want to limit the model to only see a certain number of previous
+  dialogue turns, you can set `max_history` to a finite value.
+  Please note that you should pick `max_history` carefully, so that the model has enough
+  previous dialogue turns to create a correct prediction.
```

Comment:
Should we also give the recommendation to set it to the same value as TED since we haven't really explored using them together with different settings?

Reply:
@kedz Not sure about it. What were the values that you used in your experiments to validated intentTED as an error detector of TED?

Reply:
Incorporated @JEM-Mosig 's suggestion.

Reply:
@dakshvar22 both TED and IntentTED max history was set to 6 for both SuperSTARBot and Sara.

Reply:
I see the logic behind @JEM-Mosig 's resasoning but also think the exact behavior in practice is likely to be a complicated interaction between the stories you are running intentTED on, the overall contextual dependence of your training data, the number of unique intents and their coverage in the training stories, etc, so I would be cautious about describing the behavior without doing some empirical exploration. 

An argument I see for matching histories of TED and IntentTED is that they will be making their decisions on the same history and so there wont be any cases where IntentTED fired spuriously because there was something extra it saw or something it didn't see that TED saw, but TED is operating fine. (also if we really want to commit to this, then maybe IntentTED max istory should be TED max history - 1)

Reply:
Here's the revamped description that I am thinking. Making Johannes' recommendation softer (can instead of will) but also include Chris' recommendation on using same `max_history` for `IntentTED` as for `TED`.

```
This parameter controls how much dialogue history the model looks at before making an inference.
  Default `max_history` for this policy is `None`, which means that the complete dialogue history
  since session (re)start is taken into account. If you want to limit the model
  to only see a certain number of previous
  dialogue turns, you can set `max_history` to a finite value.
  Please note that you should pick `max_history` carefully, so that the model has enough
  previous dialogue turns to create a correct prediction.
  Depending on your dataset, higher values of `max_history` can result in more frequent prediction of `action_unlikely_intent`
  as the number of unique possible conversation paths increases as more dialogue context is taken
  into account. Similarly, lowering the value of `max_history` can result in `action_unlikely_intent` being
  triggered less often but can also be a stronger indicator that the corresponding conversation path
  is highly unique and hence unexpected.
  We recommend you to set the `max_history` of `IntentTEDPolicy` equal to that of `TEDPolicy`.
```

I am advocating for equal `max_history` because max history is applied on states and not conversation turns, so `max_history - 1` will not play out very well in reality.

Reply:
@JEM-Mosig @kedz Do you agree with the refactored description above?

Reply:
Looks good to me!